src/main/java/racingcar/model/Cars.java

```diff
@@ -0,0 +1,41 @@
+package racingcar.model;
+
+import racingcar.RandomManager;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.stream.Collectors;
+
+public class Cars {
+    private static final int MOVE_CONDITION = 3;
+
+    private List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public void moveCars() {
+        this.cars.forEach(car -> car.move(isMovingForward(RandomManager.getInstance().getRandomValue())));
+    }
+
+    public static boolean isMovingForward(int value) {
+        return value > MOVE_CONDITION;
+    }
+
+    public List<Car> getWinnerCars() {
+        int maxDistance = getMaxDistance();
+        return this.cars.stream()
+                .filter(car -> car.isMaxDistance(maxDistance))
```

Comment:
```suggestion
                .filter(car -> car.isSameDistance(maxDistance))
```
이게 좀 더 자연스럽지 않나요? ㅎㅎ